hack/build-client-local.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+BIN_DIR="../../release"
```

Comment:
how about changing it to be `../../bin` and then add `bin` in the .gitignore file? @fengzixu

Reply:
Actually, we have used the dir called release to store temporary binary files. According to your comment, adding release into the .gitignore file is better. WYDT? @allencloud